CAIPs/caip-25.md

```diff
@@ -116,19 +148,20 @@ similarly, the `requiredScopes`, `optionalScopes`, and `sessionScopes` arrays re
 The same absolute security posture is not expected for the metadata objects `scopedProperties` and `sessionProperties`, but caution is still recommended for such extensions:
 callers and respondents alike SHOULD allow for their counterparties dropping or ignoring unfamiliar members from either.
 
-Requesting applications and respondents alike are expected to manage state for the connection, including `scopedProperties` and `sessionProperties`;
-if multiple concurrent connections are allowed, callers are expected to maintain them segregated and identified by the unique `sessionId` returned initially.
+When a `sessionId` is returned with the initial success response, requesting applications and respondents alike are expected to manage state for the connection, including `scopedProperties` and `sessionProperties`, and that same `sessionId` should be added to the `wallet_createSession` request to update the associated session.
```

Comment:
This paragraph contradicts my below comment: https://github.com/ChainAgnostic/CAIPs/pull/285/files#r1696794060

Reply:
see here: https://github.com/ChainAgnostic/CAIPs/pull/285#discussion_r1697158827